src/components/dashboardV2/dashboard.js

```diff
@@ -136,17 +138,27 @@ class Dashboard extends React.Component {
           </header>
 
           <div className={`${styles.main}`}>
-            <RecentTransactions />
+            <div className={styles.subContainer}>
+              {
+                isLoggedIn
+                ? <MyAccount className={styles.marginFix}/>
+                : null
+              }
+
+              <RecentTransactions className={styles.marginFix} isSignIn={isLoggedIn} />
```

Comment:
Shouldn't be better to rename the props from `isSignIn` to `isLoggedIn`? so it's consistent.
Also isSignIn, sounds like it's checking if it's the signIn flow or something like that